build/changelog/ChangelogBuilder.php

```diff
@@ -0,0 +1,149 @@
+<?php
+namespace Aws\Build\Changelog;
+
+/**
+ * @internal
+ */
+
+class ChangelogBuilder 
+{
+  
+  public $newServiceFlag = False;
+
+  public function readChangelog($dir)
```

Comment:
It looks like only `buildChangelog` and `fixEndpointFile` are called from outside the class. Every other method should be `private`.